http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/google/HttpGoogleDataprocDAO.scala

```diff
@@ -342,8 +342,10 @@ class HttpGoogleDataprocDAO(
       "spark:spark.yarn.am.memory" -> "640m"
     )
 
+    val stackdriverProps = Map("dataproc:dataproc.monitoring.stackdriver.enable" -> "true")
+
     val swConfig = new SoftwareConfig()
-      .setProperties((authProps ++ dataprocProps ++ yarnProps ++ createClusterConfig.properties).asJava)
+      .setProperties((authProps ++ dataprocProps ++ yarnProps ++ createClusterConfig.properties ++ stackdriverProps).asJava)
```

Comment:
`createClusterConfig.properties` is user-specified, I think we had put it last so it can override other configs we set by default.